docs/requirements-and-installation.md

```diff
@@ -2,9 +2,11 @@
 
```

Comment:
File already updated with same content in PR https://github.com/arista-netdevops-community/network-test-automation/pull/50. Should be updated only in one PR

Reply:
ok